library/formdata.inc.php

```diff
@@ -59,6 +59,27 @@ function escape_sort_order($s)
     return escape_identifier(strtolower($s), array("asc","desc"));
 }
 
+/**
+ * If parameter string contains comma(,) delimeter
+ * Splits parameter string into an array, using comma(,) as delimeter
+ * else it returns original string
+ * 
+ * @param   string       $s  string to be processed
+ * @return  array        $columns   an array formed by spliting $s with comma(,) delimeter         
+ */
+
+function process_cols_escape($s)
+{   
+    //returns an array of columns
+    $columns = explode(",", $s);
```

Comment:
I wonder if this is breaking because of the whitespace after the comma. Recommend going through each column in below if block and removing all leading/trailing whitespace from each item in $columns.

Reply:
yep, I confirmed this is the issue. The solution to fix this will be removing the leading/trailing spaces.

Reply:
Hello @bradymiller what am I to trim(), the elements of $columns array?
how about I do that here
`// If $s is an array process then use recursion to check each column
    if (is_array($s)) {
       $multiple_columns = [];
       foreach ($s as $column) {
           $multiple_columns[] = escape_sql_column_name(trim($column), $tables);
         }
        return implode(", ", $multiple_columns);
    }`